docs/tutorial_walk.ipynb

```diff
@@ -19,7 +19,8 @@
                 "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    A = engine.assemble_A_multi(problem, m0)
</pre><p>Why does this and the other engine methods below require problem and m0 to be input, if problem_list is already a property of engine and m0 is already a property of the problem? This seems like it could cause inconsistencies.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/symbiotic-engineering/OpenFLASH/pull/37/discussion/#comment-2463913545'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>